client/src/constants/AppConstants.js

```diff
@@ -3,3 +3,9 @@ export const GAME_STATE_NEW_GAME = 1;
 export const GAME_STATE_NEW_ROUND = 2;
 export const GAME_STATE_WAIT_FOR_NEXT_ROUND = 3;
 export const GAME_STATE_ANNOUNCE_WINNER = 4;
+export const GAME_SOUNDS = {
+  newGame: 'new-game',
```

Comment:
@existentialcoder : Can we use the capital letter convention here as well ? for eg. NEW_GAME instead of newGame.